kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -413,33 +413,39 @@ public int getInitialPartitionCount()
     return initialPartitionCount;
   }
 
-  public void setClusters(String clusters)
-  {
-    this.clusters = clusters.split(";");
-  }
-
   /**
    *  Same setting as bootstrap.servers property to KafkaConsumer
    *  refer to http://kafka.apache.org/documentation.html#newconsumerconfigs
-   *  To support multi cluster, you can have multiple bootstrap.servers separated by ";"
+   *  To support multi cluster, you can have multiple elements in the array
    */
-  public String getClusters()
+  public void setClusters(String[] clusters)
   {
-    return Joiner.on(';').join(clusters);
+    this.clusters = clusters;
```

Comment:
@tweise This issue is reported by @bhupeshchawda that the operator won't show up in gateway because it violates the BEAN convention. I'm ok with this change, as long as it is configurable through configuration file and probably test the json application also to make sure the operator  can be used everywhere